documentation/src/main/asciidoc/topics/ref_jgroups_system_properties.adoc

```diff
@@ -120,12 +120,12 @@ The following system properties apply to `default-jgroups-google.xml`:
 
 | `jgroups.google.bucket_name`
 | Name of the Google Compute Engine bucket. The name must exist and be unique.
-| N/A
-| Required if you do not set the `dns_query` parameter.
+| No default value.
+| Required
 
 |===
 
 .Reference
 
-* link:http://www.jgroups.org/manual4/index.html#SystemProperties[JGroups System Properties]
+* link:{jgroups_docs}#SystemProperties[JGroups System Properties]
 * link:http://jgroups.org/manual/html/protlist.html[JGroups Protocol List]
```

Comment:
I miss this one. can you update the URL?

Reply:
done. thanks for that. I thought we had them all too.